provision/aws/rosa_delete_cluster.sh

```diff
@@ -8,6 +8,9 @@ fi
 CLUSTER_NAME=${CLUSTER_NAME:-$(whoami)}
 if [ -z "$CLUSTER_NAME" ]; then echo "Variable CLUSTER_NAME needs to be set."; exit 1; fi
 
+# Cleanup might fail eif EFS hasn't been configured for the cluster. Ignore any failures and continue
```

Comment:
where is the input to control the EFS configuration to be set or not set ?

Reply:
EFS is the new default, and once all old clusters have been destroyed, all new will have EFS. 

I didn't want to have EFS as an option, as it makes the setup more complex.

Reply:
thanks, so line #12 in this script is pretty no-op after all the old rosa clusters are deleted, right ?

Reply:
Hm, "no-op" means to me: does nothing. I'd say it is quite the opposite: it is a no-op (does nothing) for old clusters. It only does something for the new ones with EFS. 

The current script also handles a situation when EFS creation failed for any reason, and we want to delete that part of the cluster which has been created. So I'd say I'm +1 to keep it as is for now.

Reply:
oh ok, sorry, i got it the other way around, then it makes sense, this is the catch all for any exceptions from the efs creation. thanks and it makes sense to leave it as is.